src/EditorFeatures/Core/Tagging/AggregateTagger.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Tagging;
+
+namespace Microsoft.CodeAnalysis.Editor.Tagging;
+
+/// <summary>
+/// Simple tagger that aggregates the underlying syntax/semantic compiler/analyzer taggers and presents them as
+/// a single event source and source of tags.
+/// </summary>
+internal sealed class AggregateTagger<TTag> : ITagger<TTag>, IDisposable
```

Comment:
this is jsut a move.  the type became generic as it was nested in a generic type before.